bin/kubectl-crossplane-stack-build

```diff
@@ -1,10 +1,23 @@
 #!/usr/bin/env bash
 
 function usage {
-  echo "Usage: kubectl crossplane stack [-h|--help] build [ARGS ...]" >&2
+  echo "Usage: kubectl crossplane stack [-h|--help] build [STACK_IMAGE_NAME] [ARGS ...]" >&2
   echo "" >&2
-  echo "ARGS: An alternate recipe to build, or other arguments to pass through" >&2
-  echo 'to `make`.' >&2
+  echo "" >&2
+  echo "STACK_IMAGE_NAME is the name of the stack in the local docker image" >&2
+  echo "registry to publish. It will be passed to docker, so it can start with" >&2
+  echo "a local registry address to publish to a local registry. If unspecified," >&2
```

Comment:
It seems like it might be worth providing some examples of stack image names. As a new user, I am getting a little confused reading the description of registries and images.